pkgs/development/tools/napi-rs-cli/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchurl, makeWrapper, nodejs }:
+
+stdenv.mkDerivation rec {
+  pname = "napi-rs-cli";
+  version = "2.11.4";
+
+  src = fetchurl {
+    url = "https://registry.npmjs.org/@napi-rs/cli/-/cli-${version}.tgz";
+    hash = "sha256-vZAaMh8TZ8m8oqIWwNbtcjz2GgEIE4Skpw88O0+7Z4c=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/bin"
+
+    cp scripts/index.js "$out"
```

Comment:
This is (probably?) a bad place to put this, but where should it go? `$out/share`?

Reply:
This should go in either `$out/libexec/napi-rs-cli/index.js` or
`$out/lib/napi-rs-cli/index.js` depending on whether it's executable or
not. The wrapper script stays in `$out/bin`.

Reply:
Might be better to just mimic the paths we do for other Node packages -- that is, `$out/lib/node_modules/@napi-rs/cli/scripts/index.js`? Not sure, maybe the `package.json` should be copied then.

> The wrapper script stays in `$out/bin`.

I know -- GitHub's review UI is weird, but I was only taking about that one line.

Reply:
Yarn2nix seems to put those files under `$out/libexec/$pkgname`, but either
seems fine especially since it's all internal to the package since this CLI
tool isn't a library meant to be imported.